browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -170,13 +172,20 @@ bool BraveTabContextMenuContents::IsBraveCommandIdEnabled(
       }
       return false;
     }
-    case BraveTabMenuModel::CommandShowVerticalTabs:
-      return true;
-    case BraveTabMenuModel::CommandBringAllTabsToThisWindow: {
-      return true;
-    }
     case BraveTabMenuModel::CommandCloseDuplicateTabs:
       return brave::HasDuplicateTabs(browser_);
+    case BraveTabMenuModel::CommandShowVerticalTabs:
+      [[fallthrough]];
+    case BraveTabMenuModel::CommandBringAllTabsToThisWindow:
+      [[fallthrough]];
+    case BraveTabMenuModel::CommandNewSplitView:
+      [[fallthrough]];
```

Comment:
Should we check if we can make a new split view for the tab? We can't if the tab is already tiled, right?

Reply:
Unless this can be reached asynchronosly from the tab_menu_model contstructor, I think that's unnecessary as we don't add itmes at all in case it's not possible.